_data/rajivgupta0703/rajivgupta0703.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Rajiv Kumar # No longer than 18 characters
+institution: Galgotias University 🚩 # no longer than 58 characters
```

Comment:
Remove the 🚩(flag) as it is a non-ascii character.

Reply:
resolved
I have not filled shipment details form so, please do something for that also if possible.